contracts/token/psp721/src/stub.rs

```diff
@@ -80,19 +80,6 @@ mod psp721 {
             unimplemented!()
         }
     }
-
-    #[ink(namespace = "IPSP721Mint")]
-    impl PSP721 {
-        #[ink(message)]
-        pub fn mint(&mut self, id: Id) {
-            unimplemented!()
-        }
-
-        #[ink(message)]
-        pub fn burn(&mut self, id: Id) {
-            unimplemented!()
-        }
-    }
```

Comment:
add separate mint/burn traits to the stub implementation

Reply:
it's kinda a hack here:
```rust
#[ink(namespace = "IPSP721Mint")]
    impl PSP721 {
        #[ink(message)]
        pub fn mint(&mut self, id: Id) {
            unimplemented!()
        }
    }
```

this would prefix the `mint` method in the ABI with `IPSP721Mint`. But adding these methods into the impl section is not a good practice. Better just do `impl PSP721Mint for PSP721`.

Reply:
Our `contract` macro doesn't support flags, so we can't mark it as `ink-as-dependency`.  So we can't `impl PSP721Mint for PSP721`=)